recipes/librdata/all/conanfile.py

```diff
@@ -0,0 +1,88 @@
+import os
+
+from conan import ConanFile
+from conan.tools.gnu import Autotools, AutotoolsDeps, AutotoolsToolchain
+from conan.tools.files import copy, rm, rmdir, get
+from conan.tools.apple import fix_apple_shared_install_name
+from conan.tools.microsoft import is_msvc
+from conan.tools.scm import Git
+from conan.tools.layout import basic_layout
+
+required_conan_version = ">=2.0.0"
+
+class Libreadstat(ConanFile):
+    name = "librdata"
+    version = "0.1"
+    description = "librdata is a library for read and write R data frames from C"
+    license = "MIT"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/WizardMac/librdata"
+    topics = ("r", "rdata", "rds", "data-frames")
+    package_type = "library"
+    settings = "os", "arch", "compiler", "build_type"
+    options = {
+        "shared": [True, False],
+        "fPIC": [True, False],
+        "with_zip": [True, False],
+    }
+    default_options = {
+        "shared": False,
+        "fPIC": True,
+        "with_zip": False,
+    }
+
+    def config_options(self):
+        if self.settings.os == "Windows":
+            del self.options.fPIC
+
+    def configure(self):
+        if self.options.shared:
+            self.options.rm_safe("fPIC")
+
+    def layout(self):
+        basic_layout(self, src_folder="src")
+
+    def requirements(self):
+        if self.options.with_zip:
+            self.requires("bzip2/1.0.8")
+            self.requires("zlib/1.2.13")
+            self.requires("xz_utils/5.6.3")
+
+    def source(self):
+        # git = Git(self)
+        # git.clone(url="https://github.com/WizardMac/librdata.git", target=".")
+        get(self, **self.conan_data["sources"][self.version], strip_root=True)
+
+
+    def generate(self):
+        tc = AutotoolsToolchain(self)
+        tc.generate()
+        dep = AutotoolsDeps(self)
+        dep.generate()
+
+    def build(self):
+        autotools = Autotools(self)
+        autotools.autoreconf()
+        autotools.configure()
+        autotools.make()
+
+    def package(self):
+        copy(self, pattern="LICENSE*", dst=os.path.join(self.package_folder, "licenses"), src=self.source_folder)
+        if is_msvc(self):
```

Comment:
Thanks for your comment!

I really just want to build it for MacOS and Windows instead of Linux, but even so far I haven't found a suitable way to successfully build on Windows. Always fails at the config stage.

Reply:
I could take a look during the week for windows support. Right now, I don't have access to windows machine, but I'll have by tomorrow. You can take a look in gettext recipe as example: https://github.com/conan-io/conan-center-index/blob/master/recipes/libgettext/all/conanfile.py